api/src/test/java/org/openmrs/module/referencemetadata/ReferenceMetadataActivatorComponentTest.java

```diff
@@ -45,116 +43,132 @@
 @SkipBaseSetup          // note that we skip the base setup because we don't want to include the standard test data
 public class ReferenceMetadataActivatorComponentTest extends BaseModuleContextSensitiveTest {
 
-    private ReferenceMetadataActivator activator;
-
-    @Autowired
-    private ConceptService conceptService;
-
-    @Autowired
-    private EmrApiProperties emrApiProperties;
-
-    @Autowired
-    private PatientService patientService;
```

Comment:
Are these formatting changes?

Reply:
Yes they are @dkayiwa.

Reply:
Can you remove the formatting changes?

Reply:
I have tried removing the formatting changes but I realised that the import on line 22 seems to be affecting the whole files and I seem to have just properly setup my IDE (intelliJ).

Reply:
You could just revert all changes in that file. And them carefully put them back without introducing formatting changes.

Reply:
Thanks for the feedback. I have done that.